### PR TITLE
transpose, transposed, invert, inverse, inverted, determinant, concat…

### PR DESCRIPTION
**Pull request by tomvanmele**
transpose, transposed, invert, inverse/inverted, concatenate, concatenated => method naming consistent with intended conventions.
and added determinant...

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [x] Breaking change: bug fix or new feature that involve incompatible API changes.

### Checklist

1. [ ] Add the change to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading: `Added`, `Changed`, `Removed`.
1. [x] Run all tests on your computer (i.e. `invoke test`).
1. [ ] If you add new functions/classes, check that:
   1. [ ] Are available on a second-level import, e.g. `compas.datastructures.Mesh`.
   1. [ ] Add unit tests (especially important for algorithm implementations).

### CODE REVIEW

File: src/compas/geometry/transformations/transformation.py [hunk 1]
```diff
@@ -264,11 +323,27 @@ def inverse(self):
...
         """
-        cls = type(self)
-        return cls(matrix_inverse(self.matrix))
+        # cls = type(self)
+        # return cls(matrix_inverse(self.matrix))
+        T = self.copy()
+        T.invert()
+        return T
+
+    inverted = inverse
```
Comment (romanarust) [inline src/compas/geometry/transformations/transformation.py:332]: Does `inverted` make it to the Documentation like that as well?

**Review by romanarust — COMMENTED** [review]
lgtm!

**Comment by tomvanmele** [comment]
should i merge this and bump up the version or do we want to wait with that for more changes?

**Comment by worbit** [comment]
in terms of naming i think the clearest would be if we consistently use the present/imperative verb form if it is an in-place operation, and the past tense, if a copy is returned.
`transform` > object is transformed
`transformed` > a transformed copy of object is returned, object unaffected

**Review by gonzalocasas — APPROVED** [review]
LGTM 👍

**Comment by tomvanmele** [comment]
so but merge and bump up or wait?

**Comment by gonzalocasas** [comment]
> so but merge and bump up or wait?

are there more of these changes coming? if so, wait, maybe create more PRs to merge onto this one, instead of all to master... otherwise, merge and bump...

**Pull request merged**